src/which.zig

```diff
@@ -145,3 +165,14 @@ fn whichWin(buf: *bun.WPathBuffer, path: []const u8, bin: []const u8) ?[:0]const
 
     return null;
 }
+
+test "which" {
```

Comment:
I'm excited to get zig tests working

Reply:
this was from reverting https://github.com/oven-sh/bun/pull/9691. i don't think we'll get the zigtests working